src/Compilers/Core/Portable/Syntax/GreenNode.cs

```diff
@@ -595,8 +610,11 @@ public SyntaxAnnotation[] GetAnnotations()
         {
             if (this.ContainsAnnotations)
             {
-                SyntaxAnnotation[]? annotations;
-                if (s_annotationsTable.TryGetValue(this, out annotations))
+#if NETCOREAPP
+                if (s_annotationsTable.TryGetValue(this, out var annotations))
```

Comment:
>var [](http://example.com/codeflow?start=61&length=3)

Please spell out the type.

Reply:
➡️ Changed. If this is a common item for code reviews in this path, to save time and make reviews less negative consider updating **.editorconfig** settings to reveal the situation during initial development.